[Receptionist] updates for the challenge (#930)
RoboCupAtHome/RuleBook: tasks/Receptionist.tex

The robot has to take two new guests to the living room to introduce them and offer a free place to sit.\\
    
\textbf{Main goal:}
    The robot must welcome and assist two newcomers at a party and maintain appropriate gaze direction while conversation (at person, direction of navigation).

% \vspace{-5pt}
\textbf{Optional goals:}
\begin{enumerate}[nosep]
	\item Open the entrance door for each arriving guest.

> 	\item Opening the door for the arriving guests

Reviewer: Opening the door is listed twice

	\begin{itemize}
	\item The test takes place in the living room.	
	\item The robot starts inside the \Arena{} at a predefined location.

> 	\item a table with drinks is prepared near the living room

Reviewer: ```suggestion
	\item A table with drinks is prepared near the living room
```

	
	\item \textbf{Passive Guests:} Other Guests: A few individuals, including possibly spectators, are standing in small groups of two or three, all over the arena but not covering relevant places e.g. the couch, seats or the area with the drinks.
	\end{itemize}

\end{itemize}

\subsection*{Procedure}
Both guests arrive separately. The robot either opens the door for the guest or waits for them at the starting point. It greets the guest and asks for their name. The robot then guides the guest to the beverage area, where it asks for their favorite drink and checks if the drink is available on the table and where it stands. The robot finds out the interest of the guest at a freely chosen moment. After showing the guest the beverage area, the robot escorts them to the living room and offers a free seat. Once both guests are seated, the robot introduces them to each other.
\begin{itemize}

> 	\item \textbf{Greeting guests:} the person paces a little to the left and right during conversation with the robot. other people might appear in the background

Reviewer: ```suggestion
	\item \textbf{Greeting guests:} The person paces a little to the left and right during conversation with the robot. Other people might appear in the background
```

	
	\item \textbf{Looking at person:} During verbal interactions and descriptions of people, robot 
	looks at the conversational partner. The conversational partner will make small movements to each side to confirm the robot is dynamically looking at the person. Points for looking at the person talking, will only be awarded if the robot proves to continuously look at the moving person. 
	
	\item \textbf{Looking at direction of navigation:} During navigation robot looks in the direction where it is going. Persistently gazing towards unrelated person or incorrect direction while moving during the task deducts points. 
    

>     \item \textbf{Smalltalk} ask each guest for one interest 

Reviewer: ```suggestion
    \item \textbf{Smalltalk} Ask each guest for one interest.
```

>     \item \textbf{Finding the drink} the robot shows both guest the drinking area where it will ask for their favorite drink and tell the guest if and where (left, center right is enough) that drink is available on the table. 

Reviewer: ```suggestion
    \item \textbf{Finding the drink} The robot shows both guest the drinking area where it will ask for their favorite drink and tell the guest if and where (left, center right is enough) that drink is available on the table. 
```

	
	\item \textbf{Seating People:} The robot must point at a place or location where the guest can sit.
	
	\item \textbf{Switching Places:} Guests may switch places after they were seated.
	

> 	\item \textbf{Introductions:} When introducing guests, the robot must clearly identify the person being introduced and state their name favorite drink and a interest. Introducing two people means to introduce them to each other. 

Reviewer: ```suggestion
	\item \textbf{Introductions:} When introducing guests, the robot must clearly identify the person being introduced and state their name favorite drink and an interest. Introducing two people means to introduce them to each other. 
```

	
	\item \textbf{Passive Guests:} Other Guests: A few individuals, including possibly spectators, are standing in small groups of two or three, all over the arena but not covering relevant places e.g. the couch, seats or the area with the drinks.
	\end{itemize}

\end{itemize}

\subsection*{Procedure}
Both guests arrive separately. The robot either opens the door for the guest or waits for them at the starting point. It greets the guest and asks for their name. The robot then guides the guest to the beverage area, where it asks for their favorite drink and checks if the drink is available on the table and where it stands. The robot finds out the interest of the guest at a freely chosen moment. After showing the guest the beverage area, the robot escorts them to the living room and offers a free seat. Once both guests are seated, the robot introduces them to each other.
\begin{itemize}

> 	\item \textbf{Greeting guests:} the person paces a little to the left and right during conversation with the robot. other people might appear in the background

Author: was that also discussed in the TC meeting? Cause  i was specifically told that this should be added.

> 	\item \textbf{Greeting guests:} the person paces a little to the left and right during conversation with the robot. other people might appear in the background

Reviewer 2: It was specifically requested above in order to make sure the robot keeps eye-contact. It might be enough if one person does this just to see that the robot is capable of keeping tracking on them. If we hand out post-its with Name, Drink, Interest, it should be fine I think.

> 	\item \textbf{Greeting guests:} the person paces a little to the left and right during conversation with the robot. other people might appear in the background

Reviewer 3: what we talked in the TC meeting is that is must be added one way or the other, a lot of robots don't do this and many times robots are talking to a person, the person moves aw bit and the robot keeps looking to an empty space. This just seems very weird. As @hawkina said, doing this with one person is enough to understand if the robot does dynamic tracking or not.

	looks at the conversational partner. During navigation robot looks in the direction where it is going. Persistently gazing towards unrelated 
	person or incorrect direction while moving during the task deducts points.
    

>     \item \textbf{Smalltalk} ask each guest after one interest 

Reviewer 3: I think you missed this edit...
Should be something like:
\item \textbf{Smalltalk} ask each guest one interest during guidance or at the drinking area

Author: no i actually removed that on purpose because i would not mind asking the guest in the living room for one interest. However if you want it changed back i could do that

Reviewer 3: oh, ok for me no problem. Then i would just replace the "after " by "for" so the sentence is semantically correct.

Author: oh right, thanks :)

The referees need to:
\begin{itemize}

> 	\item Provide the list of names and drinks on setup day

Reviewer: ```suggestion
	
```

	\item Assign name, drink and one interest to 3 volunteers.
	\item Arrange (and re-arrange) people in the living room.

> 	\item change the selection in the beverage area

Reviewer: ```suggestion
	\item Change the selection in the beverage area.
```

	\item Open the door when requested by the robot.
\end{itemize}

\subsubsection*{To OC}

During setup day:
\begin{itemize}

> 	\item announce beverage location

Reviewer: ```suggestion
	\item Announce beverage location.
```

> 	\item the list of names and drinks is given (a list of interests will not be provided)

Reviewer: Names is covered under rule:scenario_names, drinks are a defined category.
You can remove this.
```suggestion
	
```

The referees need to:
\begin{itemize}
	\item Assign name, drink and one interest to 3 volunteers.
	\item Arrange (and re-arrange) people in the living room.
	\item change the selection in the beverage area
	\item Open the door when requested by the robot.
\end{itemize}

\subsubsection*{To OC}

During setup day:
\begin{itemize}

> 	\item  the list of names and drinks is given (a list of interests will not be provided)

Reviewer: This should also be a TODO list for referees:

```suggestion
        \item Provide the list of names and drinks.
```

> \end{itemize}

Reviewer: This misses `announce beverage location`

At least two hours before test:
\begin{itemize}
	\item Announce starting position.
	\item Announce host's name, favorite drink and interest.

> 	\item Recruit volunteers as host and two guests and passive guests .

Reviewer: May add numbers here `Recruit x volunteers. (job listings)` ?
